### PR TITLE
Add queryFeeDetails

### PR DESCRIPTION
**Pull request by imstar15**
```
yarn start

...
2. Query automationTime fee details 
automationFeeDetails:  { executionFee: '722.4000 mTUR', xcmpFee: '1.5668 TUR' }
...
```

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -108,19 +108,23 @@ async function main() {
...
   console.log('xcmpCall: ', xcmpCall);
 
-  console.log(`\n2. Estimating XCM fees ...`);
+  console.log(`\n2. Query automationTime fee details `);
+  const { executionFee, xcmpFee } = await turingHelper.api.rpc.automationTime.queryFeeDetails(xcmpCall);
+  console.log(`automationFeeDetails: `, { executionFee: executionFee.toHuman(), xcmpFee: xcmpFee.toHuman() });
+
+  console.log(`\n3. Estimating XCM fees ...`);
   const xcmFrees = await turingHelper.getXcmFees(turingAddress, xcmpCall);
   console.log("xcmFrees:", xcmFrees.toHuman());
```
Comment (chrisli30) [inline src/index.js:117]: the result xcmpFee from `queryFeeDetails` is supposed to replace this `getXcmFees`, correct?

@nvengal

**Review by chrisli30 — COMMENTED** [review]
Added one comment for Nikhil to take a look.

File: src/index.js [hunk 2]
```diff
@@ -108,19 +108,23 @@ async function main() {
```
Comment (chrisli30) [inline src/index.js:108]: @imstar15 
Please replace the time parameter with actual timestamps,
    `Fixed: { executionTimes: [0] }`

**Review by chrisli30 — CHANGES_REQUESTED** [review]

File: src/index.js (hunk 2 above)
Comment (chrisli30) [reply src/index.js:108]: Never mind, [0] is to execute immediately. No need to change.

File: src/index.js (hunk 1 above)
Comment (nvengal) [reply src/index.js:117]: Yes, that's correct. `getXcmFee` is the old version.

Comment (chrisli30) [reply src/index.js:117]: okay, Charles let’s remove the old `xcmFrees` variable and use the one returned from queryFeedDetails.

**Review by chrisli30 — APPROVED** [review]

File: package.json [hunk 3]
```diff
@@ -5,8 +5,8 @@
     "lint": "eslint --fix src/*"
   },
   "dependencies": {
-    "@imstar15/api-augment": "^0.0.12",
-    "@imstar15/types": "^0.0.12",
+    "@imstar15/api-augment": "^0.0.14",
+    "@imstar15/types": "^0.0.14",
```
Comment (chrisli30) [inline package.json:9]: Let’s publish the new code of @oak-network/api-augment and reference it here.

Comment (chrisli30) [reply package.json:9]: to replace your personal library

File: src/index.js (hunk 1 above)
Comment (chrisli30) [reply src/index.js:117]: @imstar15 this line still needs update.

**Review by chrisli30 — CHANGES_REQUESTED** [review]

File: package.json (hunk 3 above)
Comment (nvengal) [reply package.json:9]: Looks good other than using the personal lib here.

**Review by nvengal — COMMENTED** [review]
Looks good other than using the personal lib being used (like chris commented, https://github.com/OAK-Foundation/xcm-demo/pull/8/files#r1051080922)

File: package.json (hunk 3 above)
Comment (imstar15) [reply package.json]: Fixed

**Review by chrisli30 — APPROVED** [review]
okay great looks good!

**Pull request merged**